### DISCUSSION TITLE
How to use the build version of assets instead of running `npm run dev`

### DISCUSSION THREAD

**Discussion by amoktar** [laravel/framework#50221] [Q&A] [answered]
Hello guys 
How to use the build version of assets instead of running `npm run dev` 

I'm working on project and I don't need to run vite serve , since I'm not changing any assets .
I'm using @vite(['resources/css/app.css', 'resources/js/app.js']) but in the source code it tranlate that to http://[::1]:5173/@vite/client

**Comment by henzeb** [comment]
npm run build?

**Reply by amoktar** [reply]
I'm using @Vite(['resources/css/app.css', 'resources/js/app.js']) but in the source code it tranlate that to http://[::1]:5173/@vite/client

How can I make this to reference the build files

**Reply by henzeb** [reply]
Does my command alternative not work?

**Reply by amoktar** [reply]
Thanks for response. It still tranlate assets to http://[::1]:5173/@vite/client

**Comment by amoktar** [comment, marked as answer]
The answer is 
Delete `/public/hot` file
